### ISSUE TITLE
Turn on Clone Cell Across Dashboards

### ISSUE THREAD

**Issue by jzahnd** [influxdata/ui#3678]
We would like to turn on Clone Cells Across Dashboards. Talk with Product before turn that on.

## Acceptance Criteria
- Turn Clone Cell Across Dashboards and verify in Production.
- Create a task for removing the flag in a month

**Comment by jzahnd** [comment]
We do indeed want to turn on Clone Cell Across Dashboards, there will be a design update needed that will follow in a separate issue.

**Comment by jzahnd** [comment]
@asalem1 The current consensus is that we are comfortable pushing this before the design update. That being said, @juliajames12 do we have an idea of when this design update will be settled / proposed?

**Comment by jzahnd** [comment]
@juliajames12 bump, any idea about design?

**Comment by juliajames12** [comment]
Here are the updates I'd like us to make for this experience (feel free to split out into separate tickets):

- Change name of action to "Move" instead of "Relocate"
- Update title of modal to "Move or Copy Cell to Dashboard"
- Change Move/Copy selection to switch buttons instead of toggle and move to top of modal
- Update the Dashboard selection to include "Dashboard" label, subtext "Where do you want to move/copy your cell to?", and "Choose Dashboard" placeholder text

Before:
<img width="125" alt="Menu Item (Relocate)" src="https://user-images.githubusercontent.com/81173353/153626740-c37f2aa2-b3ce-406c-a314-372fefcab140.png">
<img width="486" alt="Move Cell Modal" src="https://user-images.githubusercontent.com/81173353/153626765-c5f55946-aa30-4eeb-8440-b287d3dc4622.png">

After:
<img width="125" alt="Menu Item (Move)" src="https://user-images.githubusercontent.com/81173353/153626821-1749eef9-5a83-413e-8c39-7bbcf196c0d9.png">
<img width="468" alt="Modal (Move Selected)" src="https://user-images.githubusercontent.com/81173353/153626847-4a988d27-f18a-4306-9ba7-93fdc44ec30b.png">
<img width="468" alt="Modal (Copy Selected)" src="https://user-images.githubusercontent.com/81173353/153626866-7678c76d-a855-4eff-8a71-edc8bbd9b927.png">

Figma: https://www.figma.com/file/8RawU20Waig4v8BhEbjtSr/Clone-Cell?node-id=3%3A415

**Comment by jzahnd** [comment]
This is currently blocked by #3678, when the design updates (also as a comment above) are complete we can move this task forward.

**Comment by garylfowler** [comment]
This has product approval to turn on in production when ready.

**Comment by appletreeisyellow** [comment]
This flag is turned on and verified in production.
